### PR TITLE
update README

### LINKED ISSUE #45: Update README

### PR DESCRIPTION
**Pull request by rengelke**
Close #45

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -66,8 +92,71 @@ compound("path/to/Alfentanil-Model.json")
 
 ### Explore Compound
 
-```{r}
-rifampicin$Formulations
+Once you have imported a compound, you can explore its properties, such as formulations, protocols, and parameters:
```
Comment (dlefaudeux) [inline README.Rmd:95]:
```suggestion
Once you have imported a compound model, you can explore its properties, such as formulations, protocols, and parameters:
```

File: README.Rmd [hunk 2]
```diff
@@ -66,8 +92,71 @@ compound("path/to/Alfentanil-Model.json")
...
+
+```{r, include=FALSE}
+# Explore the formulations of Rifampicin
+rifampicin$formulations
+```
+
+This command outputs the formulation details of the compound, allowing you to understand how the compound is configured in the simulation.
```
Comment (dlefaudeux) [inline README.Rmd:102]:
```suggestion
This command outputs the formulations that are defined for the compound and that can be used in simulations.
```

**Review by dlefaudeux — APPROVED** [review]
All good with me. 
2 minor suggestions.

File: README.Rmd (hunk 1 above)
Comment (Felixmil) [reply README.Rmd:95]: I would replace "model" by "snapshot" in the sentence.

**Review by Felixmil — APPROVED** [review]

**Pull request merged**